recipes/lz4/lz4_r131.oe

```diff
@@ -0,0 +1,5 @@
+require ${PN}.inc
+
+SRC_URI += "file://windows-h-include.patch"
+SRC_URI += "file://exeext.patch"
```

Comment:
Good question.

Pro: They are only relevant for mingw.

Con: Developers not doing mingw build themselves will see patch conflicts if we apply mingw (and similar for osx patches) for linux builds also.

Let me know what you want here.